### PR TITLE
fix CVE-2022-41915

### PR DESCRIPTION
**Pull request by alanlvle**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

File: apm-application-toolkit/apm-toolkit-kafka/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>apm-application-toolkit</artifactId>
         <groupId>org.apache.skywalking</groupId>
-        <version>8.13.0-SNAPSHOT</version>
```
Comment (wu-sheng) [inline apm-application-toolkit/apm-toolkit-kafka/pom.xml:24]: Why do you change all versions to a release status?

Comment (alanlvle) [reply apm-application-toolkit/apm-toolkit-kafka/pom.xml:24]: I am sorry,  I create branch from tag v8.13.0

Comment (wu-sheng) [reply apm-application-toolkit/apm-toolkit-kafka/pom.xml:24]: That is wrong. We only accept PR from a dev version. You need to update from the main branch.

Comment (alanlvle) [reply apm-application-toolkit/apm-toolkit-kafka/pom.xml:24]: ok

**Pull request closed without merging**